docs/manual/common/concepts/MicroserviceSystemDesign.md

```diff
@@ -1,7 +1,27 @@
 # Designing your microservices system
 
-Bonér's [Reactive Microservices Architecture: Design Principles for Distributed Systems](http://www.oreilly.com/programming/free/reactive-microservices-architecture.html) is fundamental reading for microservices systems architects. It takes time and experience to become skilled in this complex domain. Lagom provides practical guidance by offering opinionated APIs, implementations of supporting features, and appropriate defaults.
+Bonér's [Reactive Microservices Architecture: Design Principles for Distributed Systems](http://www.oreilly.com/programming/free/reactive-microservices-architecture.html) is fundamental reading for microservices systems architects. It takes time and experience to become skilled in this complex domain. Lagom provides practical guidance by offering opinionated APIs, implementations of supporting features, and appropriate defaults. 
 
-If you are creating microservices to replace an existing monolith, you might need to vary your approach based on its architecture. For example, if it is reasonably well-designed, you might be able to first de-couple its components and focus on migrating functionality a bit at a time. Whether that is the case or not, we recommend starting small. First, identify a need for a simple microservice that can consume asynchronous messages. It needn't be complex or even provide a lot of value. The simplicity reduces the risk associated with deployment and can provide a quick win. Then, at the architectural level pull out a core service that can be compartmentalized, and decide how to divide it into microservices. When you attack the problem a piece at a time, you and your team will learn as you go and will become increasingly effective.
+We recommend starting small. First, identify a need for a simple microservice that can consume asynchronous messages. It needn't be complex or even provide a lot of value. The simplicity reduces the risk associated with deployment and can provide a quick win. Next, at the architectural level, pull out a core service that can be compartmentalized. Divide it into a system of microservices. When you attack the problem a piece at a time, you and your team will learn as you go and will become increasingly effective. Employing approaches such as [Domain-driven Design](https://en.wikipedia.org/wiki/Domain-driven_design) (DDD) can help your organization deal with the complexity inherent in enterprise systems.
```

Comment:
These sentences only make sense to me in the context of replacing an existing system:

* "First, identify a need for a simple microservice that can consume asynchronous messages." — for a service to consume messages, something needs to be producing them
* "Next, at the architectural level, pull out a core service that can be compartmentalized." — "pull out" from what?